ecosystem/indexer-grpc/indexer-grpc-fullnode/src/convert.rs

```diff
@@ -241,6 +241,7 @@ pub fn convert_move_type(move_type: &MoveType) -> transaction::MoveType {
         MoveType::Struct(_) => transaction::MoveTypes::Struct,
         MoveType::GenericTypeParam { .. } => transaction::MoveTypes::GenericTypeParam,
         MoveType::Reference { .. } => transaction::MoveTypes::Reference,
+        MoveType::Function { .. } => transaction::MoveTypes::Unparsable,
```

Comment:
Is this `transaction::MoveTypes` yet another type? 🤦‍♂️

Reply:
Yeah. I asked at the indexer channel, for now no answer. This is some protobuf stuff, and I do not known what's it used for. I discovered it also doesn't support enums, no one complained so far...